spotirec.py

```diff
@@ -466,11 +469,11 @@ def print_devices(save_prompt=True, selection_prompt=True):
     :param selection_prompt: whether or not user should be prompted for a selection
     """
     devices = api.get_available_devices(headers)['devices']
-    print('Available devices:')
-    print(f'Name{" " * 19}Type')
-    print("-" * 40)
+    logging.info('Available devices:')
+    logging.info(f'Name{" " * 19}Type')
+    logging.info("-" * 40)
     for x in devices:
-        print(f'{devices.index(x)}. {x["name"]}{" " * (20 - len(x["name"]))}{x["type"]}')
+        logging.info(f'{devices.index(x)}. {x["name"]}{" " * (20 - len(x["name"]))}{x["type"]}')
```

Comment:
This is also related to a prompt, so should also probably be printed no matter what.

Reply:
Fixed in d3df488